page.php

```diff
@@ -10,18 +10,18 @@
  * @since FoundationPress 1.0.0
  */
 
- get_header(); ?>
+get_header(); ?>
 
- <?php get_template_part( 'template-parts/featured-image' ); ?>
- <div class="main-container">
+<?php get_template_part( 'template-parts/featured-image' ); ?>
+<div class="main-container">
     <div class="main-grid">
-     <main class="main-content">
-    	 <?php while ( have_posts() ) : the_post(); ?>
-    	 	<?php get_template_part( 'template-parts/content', 'page' ); ?>
-    		<?php comments_template(); ?>
-    	 <?php endwhile;?>
-     </main>
-    <?php get_sidebar(); ?>
+        <main class="main-content"[>
```

Comment:
@giollianosulit it looks like you accidentally added a left bracket on this line?